ush/wave_outp_spec.sh

```diff
@@ -214,13 +175,7 @@
      fi
    fi
   else
-    set +x
-    echo ' '
-    echo '***************************************************************** '
-    echo "*** FATAL ERROR : OUTPUT DATA FILE FOR BOUY ${bouy} NOT FOUND *** "
-    echo '***************************************************************** '
-    echo ' '
-    set_trace
+    echo "ERROR: OUTPUT DATA FILE FOR BOUY ${bouy} NOT FOUND"
     exit 5
```

Comment:
```suggestion
    err_chk "FATAL ERROR: OUTPUT DATA FILE FOR BOUY ${buoy} NOT FOUND"
```

Reply:
I saw a few instances of the `${buoy}` spelling incorrectly in the echos. Double check for others I may have missed.